chord_metadata_service/chord/ingest/phenopackets.py

```diff
@@ -376,19 +375,16 @@ def ingest_phenopacket(phenopacket_data: dict[str, Any],
     # Attach resources to the metadata object
     meta_data_obj.resources.set(resources_db)
 
-    # Create the phenopacket object...
-    new_phenopacket = pm.Phenopacket(
-        id=new_phenopacket_id,
+    # Get the phenopacket if one exists for given ID, or create a new one.
+    new_phenopacket, created = pm.Phenopacket.objects.get_or_create(
```

Comment:
not necessarily a `new_phenopacket` anymore (except, it will actually always be new - see the review comment; but then the comment here is wrong since it won't get that phenopacket itll just error) - also `created` is never used